parsec/core/fs/userfs/userfs.py

```diff
@@ -201,10 +202,15 @@ def __init__(
             role_cached_on=now,
             role=WorkspaceRole.OWNER,
         )
+
+        async def _get_previous_entry() -> NoReturn:
```

Comment:
I wouldn't use a `NoReturn` here given UserRemoteLoader's `get_previous_entry` is supposed to return the entry info, it's just * this * implementation that is not functional

If we want to sugnify that `get_previous_entry` is never going to be called for this instance (because userfs is never going to be reencrypted), then we should raise a `RuntimeError` or `AssertionError` instead of the `NotImplementedError`

Reply:
9ddb38c5be108a3fc89d6ff4ab2404d92401c0ed